CHANGES.md

```diff
@@ -1,4 +1,30 @@
-## edge-19-2.5
+## edge-19.3.1
+
+* CLI
+  * Introduce a check for NET_ADMIN in `linkerd check`
+  * Fixed permissions check for CRDs
+  * Include kubectl version check as part of `linkerd check` (thanks @yb172!)
```

Comment:
Some of these leading verbs are present tense and some are past tense. Mind standardizing on past tense?

```suggestion
  * Included kubectl version check as part of `linkerd check` (thanks @yb172!)
```

Same goes for "_Introduced_ a check for NET_ADMIN" above, and some of the other bullets below.